src/style-dictionary-dist2/TerrawareTheme2.ts

```diff
@@ -0,0 +1,533 @@
+import { PaletteOptions } from '@mui/material/styles/createPalette';
+
+declare module '@mui/material/styles/createPalette' {
+  export interface Palette {
+    TwClrBaseBlue050?: React.CSSProperties['color'];
```

Comment:
should we maybe make these hyphenated?
tw-clr-base-blue-100?

Reply:
Hmm, that does make sense for scss variables, but camelCase is more standard for TS names isn't it?

Reply:
was thinking about how we might end up using the tokens in terraware-web:
`theme.palette.TwClrBaseBlue050` vs `theme.palette.tw-clr-base-blue-050`
I guess we could start with the way it is here and evolve as we get feedback from all devs

Reply:
actually, what you have written is more readable I think. I'll try getting it into that format.

Reply:
i.e. for this particular case, of referencing styles, kebab-case is a bit less friction, probably

Reply:
i do agree with you on readability and ease-of-use